scripts/snapshot.py

```diff
@@ -83,42 +81,41 @@ def validate_options(options):
     options.source_repo = apparent_source_repo_type
   else:
     if options.source_repo != "none" and options.source_repo != apparent_source_repo_type:
-      raise RuntimeError("Specified source repository type of %s conflicts with determined type of %s" % \
-        (options.source_repo, apparent_source_repo_type))
+      raise RuntimeError(f"Specified source repository type of {options.source_repo} conflicts with determined type of {apparent_source_repo_type}")
 
   if options.dest_repo == "":
     #destination repo type is not specified to just using the apparent type.
     options.dest_repo = apparent_dest_repo_type
   else:
     if options.dest_repo != "none" and options.dest_repo != apparent_dest_repo_type:
-      raise RuntimeError("Specified destination repository type of %s conflicts with determined type of %s" % \
-        (options.dest_repo, apparent_dest_repo_type))
+      raise RuntimeError(f"Specified destination repository type of {options.dest_repo} conflicts with determined type of {apparent_dest_repo_type}")
 
   return options
 #end validate_options
 
 def run_cmd(cmd, options, working_dir="."):
   cmd_str = " ".join(cmd)
   if options.verbose_mode:
-    print "Running command '%s' in dir %s." % (cmd_str, working_dir)
+    print (f"Running command '{cmd_str}' in dir {working_dir}")
```

Comment:
There was a `.` here
```suggestion
    print (f"Running command '{cmd_str}' in dir {working_dir}.")
```

Reply:
I thought the `.` made the readability a bit messy, for example when the working_dir is `.` we get a `. .` output, which I thought was a bit messy to read. If you feel strongly I can re-add, but my preference would be to remove for readability improvement

Reply:
I didn't think of it, yeah it makes sense for it to be removed then